docs/change-log/2025-07-10-clickable-links-and-customizable-statuses-in-activities.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Clickable Links and Customizable Statuses in Activities"
+date: "2025-07-10"
+topics:
+- "Activities"
+- "Discord Social SDK"
+- "Embedded App SDK"
+---
+
+We've added new functionality to Rich Presences to give users of your application a more interactive and flexible experience. There are two big changes as part of this:
+- You can now add clickable links to the state text, details text, large image & small image
+- You can now choose which field (name, state, or details) is used in users' status text in the member list (e.x. instead of "Listening to MyMusic" you can now have your status text show "Listening to Rick Astley")
```

Comment:
```suggestion
- You can now choose which field (name, state, or details) is used in users' status text in the member list (e.g. instead of "Listening to MyMusic" you can now have your status text show "Listening to Rick Astley")
```